cogs/events.py

```diff
@@ -1,11 +1,23 @@
 import discord
 
-class Events():
+import config
+
+
+class Events:
     def __init__(self, bot):
         self.bot = bot
+    
+    async def on_message(self, message):
+        print(message.server.id)
+    
+    async def on_message_edit(self, before, after):
+        pass
+    
+    async def on_message_delete(self, message):
+        log_channel = self.bot.get_channel(config.guilds[message.server.id]['server-log'])
```

Comment:
* You could make this global to the class. 
* Why doesn't the bot automatically find and make one if it does not exist? Could that be a command?